src/actions/facebookcustomermatch/lib/util.ts

```diff
@@ -0,0 +1,85 @@
+const countries:any = require("i18n-iso-countries")
+const usStates:any = require('us')
+const sugarDate:any = require("sugar").Date
```

Comment:
These packages will need to the added to package.json (`yarn add i18n-iso-countries` etc)

Unfortunately `us` does not have a license we can use in this open source repo (Apache 2.0, MIT, or BSD (2-clause or 3-clause) licenses are OK). Could we replace this with a simple lookup for state abbreviations and names?